js/bybit.js

```diff
@@ -0,0 +1,1001 @@
+        await this.loadMarkets ();
+        const request = {};
+        const response = await this.wapiGetPositionList (this.extend (request, params));
+        const retData = this.safeValue (response, 'result');
+        const result = { 'info': retData };
+        for (let i = 0; i < retData.length; i++) {
+            const position = retData[i];
+            const symbol = this.safeString (position, 'symbol');
+            const market = this.markets_by_id[symbol];
+            const currencyId = market['base'];
+            const code = this.safeCurrencyCode (currencyId);
+            const account = this.account ();
+            account['total'] = position['wallet_balance'];
+            account['used'] = position['position_margin'] + position['occ_closing_fee'] + position['occ_funding_fee'] + position['order_margin'];
+            result[code] = account;
+        }
+        return this.parseBalance (result);
+    }
+
+    async fetchOHLCV (symbol, timeframe = '1m', since = undefined, limit = undefined, params = {}) {
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const request = {
+            'symbol': market['id'],
+            'interval': this.timeframes[timeframe],
+        };
+        if (since === undefined) {
+            request['from'] = this.seconds () - 86400; // default from 24 hours ago
+        } else {
+            request['from'] = this.truncate (since / 1000, 0);
+        }
+        if (limit !== undefined) {
+            request['limit'] = limit; // default == max == 200
+        }
+        const response = await this.v2GetPublicKlineList (this.extend (request, params));
+        return this.parseOHLCVs (this.safeValue (response, 'result', {}), market, timeframe, since, limit);
+    }
+
+    async fetchOrder (id, symbol = undefined, params = {}) {
+        // There are two type of orders, so user must define which order to cancel
+        // so why not ask user to point out in  which order to cancel ?
+        let orders = undefined;
+        if (('stop_order_id' in params) || ('order_id' in params)) {
+            orders = await this.fetchOrders (symbol, undefined, undefined, params);
+        } else {
+            throw new ArgumentsRequired (this.id + " fetchOrder() requires additional argement either 'order_id' to fetch active order or 'stop_order_id' to fetch conditional order. Values of these arguments should be equal to value of 'id' argument.");
+        }
+        const numResults = orders.length;
+        if (numResults === 1) {
+            return orders[0];
+        } else {
+            for (let i = 0; i < numResults; i++) {
+                if (this.safeString (orders[i], 'id') === id) {
+                    return orders[i];
+                }
+            }
+        }
+        throw new OrderNotFound (this.id + ': The order ' + id + ' not found.');
+    }
+
+    async fetchOpenOrders (symbol = undefined, since = undefined, limit = undefined, params = {}) {
```

Comment:
Both `fetchOpenOrders` and `fetchClosedOrders` should be removed, if the only endpoint provided by the exchange is the one that corresponds to `fetchOrders`. The user can do the filtering on his side in this case.